trie/trieStorageManagerInEpoch_test.go

```diff
@@ -1,21 +1,24 @@
 package trie
 
 import (
+	"errors"
 	"strings"
 	"testing"
 
-	"github.com/multiversx/mx-chain-core-go/core/check"
+	errorsMx "github.com/multiversx/mx-chain-go/errors"
```

Comment:
`internalErrors`

Reply:
decided to use errorsMx in a previous PR and kept the same everywhere